code/__HELPERS/pronouns.dm

```diff
@@ -1,5 +1,12 @@
+#define GET_TARGET_PRONOUN(target, pronoun) call(target, ALL_PRONOUNS[pronoun])(null)
+#define GET_GENERIC_PRONOUN(pronoun) call(/datum/, ALL_PRONOUNS[pronoun])(null)
```

Comment:
If I am understanding this correctly: 

I do not think there will be any universe where people use `GET_GENERIC_PRONOUN` - all the generic pronouns just return... the pronoun you're already typing. IE, `GET_GENERIC_PRONOUN(%PRONOUN_their)` will always result in just printing `their` anyways

Reply:
You're right for generic pronouns; the intent was to facilitate people making text with the appropriate gendered pronouns for procedurally made conversation or text. I'll change it up to facilitate that instead.